kimchi/src/circuits/expr.rs

```diff
@@ -51,15 +51,36 @@ pub enum ExprError<Column> {
     MissingRuntime,
 }
 
-pub struct Challenges<F> {
+pub struct BerkeleyChallenges<F> {
     /// The challenge alpha from the PLONK IOP.
     pub alpha: F,
     /// The challenge beta from the PLONK IOP.
     pub beta: F,
     /// The challenge gamma from the PLONK IOP.
     pub gamma: F,
     /// The challenge joint_combiner which is used to combine joint lookup tables.
-    pub joint_combiner: Option<F>,
```

Comment:
Sometimes, the joint combiner is non-existent, for instance when we do have a single column.

Reply:
see previous comment, it's ok to make that change
It is also necessarry to index the challenge term